docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "Kees Smit"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The CMUcam5 Pixy camera is available in a version designed for
+LEGO Mindstorms. In this tutorial we explain how to use it with ev3dev
+and python.
+General information about Pixy for LEGO MIndstorms can be found on
+Charmed Lab's [wiki-page](http://cmucam.org/projects/cmucam5/wiki/LEGO_Wiki).
+
+## What you need
+
+* LEGO MIndstorms EV3.
+* CMUcam5 Pixy for LEGO Mindstorms, connected to the EV3 with the cable
+deliverd with your camera. If you have a regular version of CMUcam5 Pixy
+you will need a special Pixy adapter for EV3.
+
+**Note:** the Pixy camera comes with it's own tool: PixyMon. This tool helps
```

Comment:
If you want this note to stand out more, you can make it into an info or warning box.

```
{% include /style/icon.html type="warning" %} Example. Text goes here. {:.alert .alert-warning}
```